documentation/documentation_publisher.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-console */
+import AWS from 'aws-sdk';
+import {Command} from 'commander';
```

Comment:
Nit, I know `coda` just eliminated yargs in favor of Commander, but we only use yargs in this repo right now. Would be nice to migrate the the whole repo to Commander, but we have a lot bigger fish to fry right now, so would suggest just using yargs so we don't have to deal with two libraries for commands, and we can migrate the whole repo all at once at some later date.

Reply:
Yeah, these things tend to sit in an intermediate state for months. It's not great if you're trying to make simple changes to logic and you find yourself having to learn a whole new library just because the repo is inconsistent.

Reply:
ok done